pypeplot

```diff
@@ -2,16 +2,24 @@
 
 # Alex Cornejo <acornejo@gmail.com> 2013
 # Source released under MIT license
+#
+# Forked by Stipe Galić (stipe.galic@gmail.com)
```

Comment:
Since I want to merge your changes, this comment no longer applies. Would you mind changing it to something along the lines of:

Contributors:
  - Stipe Galic <stipe.galic@gmail.com>